Makefile

```diff
@@ -538,6 +538,7 @@ check-symbols: startup_files libc
 	    -U__GNUC_PATCHLEVEL__ \
 	    -U__VERSION__ \
 	    -U__FLOAT128__ \
+	    -U__NO_MATH_ERRNO__ \
```

Comment:
Maybe add a TODO above to go with the others... and maybe we should fix those TODOs ?

Reply:
There aren't TODOs for the others, though. And I'm not sure what there would be to actually do here...

Reply:
For example `TODO: Undefine __FLOAT128__ for now since it's not in clang 8.0.` above.. it sounds like we no longer need to undefined that macro because we don't need to support llvm 8.0.0 anymore.

Basically some of these we exclusions we want keep forever and other are there to ease the transition from on llvm version to another.  In this case I believe that this is just a transitory addition, right?  Once we require llvm 14 we can remove this line?